src/Components/Dropdowns/Dropdown.scss

```diff
@@ -69,10 +72,18 @@ $dropdown-item--padding-v: $cf-space-2xs;
 .cf-dropdown-divider {
   @include dropdownItemStyles();
   font-weight: $cf-font-weight--medium;
+  text-transform: uppercase;
+  color: #88889b;
```

Comment:
Since this is just a divider - does `text-transform: uppercase` have any effect?

Reply:
Gotcha - thanks!